code/index.html

```diff
@@ -2,29 +2,182 @@
+
+      <h3>Would you like to add something?</h3>
+
+      <textarea class="textarea" placeholder="Write here" rows="4" cols="30"></textarea>
+
+      <br />
+      <br />
+
+        <label for="checkbox"> 
+          <input type="checkbox" id="checkbox" name="checkbox" required> I agree to <a href="#">terms & conditions</a>.</label>
+        
+          <br />
+          <br />
+        
+          <button type="submit">Order today</button>
+      </div>
+      
+      </form>
+ 
+
+    <!-- ACCORDION FAQ -->
+    <section class="accordion">
+       
+        <div class="big-header">FAQ</div>
+    
+        <div class="question" id="section1">About Le Fleur <img class="arrow" src="./logo/down-arrow.png">
+        </div>
+            <div class="answer">
+              <p>Le Fleur was established so that you can buy flowers directly from a community of trusted local florists.</p> 
+            <br />
+            <p>We're proud to have been connecting customers with the nation's best florists for more than 50 years. Headquartered in Stockholm, Sweden, Le Fleur has over 10 000 member florists in the network.</p>
+              
+            </div>
+
+        <div class="question" id="section2">I need to get in contact with you! <img class="arrow" src="./logo/down-arrow.png"></div>
+            <div class="answer">
+              <p>Please send us an email 
+              <br />
+            
+              💌 <a href="mailto:hello@lefleur.com">hello@lefleur.com</a></p>
+            </div>
+              
+        <div class="question" id="section3">What if I change my mind? <img class="arrow" src="./logo/down-arrow.png"></div>
+            <div class="answer">
+              <p>Call us and we're happy to help! 
+              <br /> 
+              
+              ☎️ <a href="tel:0046-123-456-789">123-456-789</a></p>
+            </div>
+             
+        <div class="question" id="section4">Can I pick a delivery time? <img class="arrow" src="./logo/down-arrow.png"></div>
+            <div class="answer">
+              <p>As soon as we have recieved an order confirmation from our supplier you will get a text message with all the details.</p>
+              
+              <br />
+              
+              <p>You will then be able to choose a delivery time and provide us with more information. </p>
+        </div>
+
     </section>
-    <!-- dont forget to link the javascript file here! -->
+
+    </main>
+      
+      <footer>
+          <p>© Made by <a href="https://github.com/emmajosefina/">Emma Lindell</a></p>
+          
+            <br />
+
+          <p>Technigo Bootcamp 2022 – Project 3 – Business Site.</p>
+
+          <!-- SOCIAL MEDIA ICONS-->
+          <p>
+            <a href="#" class="fa fa-lg fa-facebook"></a>
+            <a href="https://www.linkedin.com/in/emmalindell4/" class="fa fa-lg fa-linkedin"></a>
+            <a href="https://github.com/emmajosefina" class="fa fa-lg fa-github"></a>
+          </p>
```

Comment:
Beautiful footer!! Nice that you linked you linkedIn in the footer I will also start doing that